test/models/track_test.rb

```diff
@@ -16,6 +16,14 @@ class TrackTest < ActiveSupport::TestCase
     assert_equal track, Track.for!(track.slug)
   end
 
+  test "#active scope" do
```

Comment:
What do you think about using `.active` instead? It's a class method.

Reply:
```suggestion
  test ".active scope" do
```